pkg/controller/disruption/disruption.go

```diff
@@ -134,6 +134,7 @@ type controllerAndScale struct {
 type podControllerFinder func(ctx context.Context, controllerRef *metav1.OwnerReference, namespace string) (*controllerAndScale, error)
 
 func NewDisruptionController(
+	ctx context.Context,
```

Comment:
How can Kubernetes compile when the function requires a new parameter and some callers of this function aren't getting updated?

/test pull-kubernetes-verify-strict-lint